lib/Runtime/Library/SIMDInt8x16Lib.cpp

```diff
@@ -805,7 +805,7 @@ namespace Js
         Assert(!(callInfo.Flags & CallFlags_New));
 
 
-        return SIMD128TypedArrayLoad<JavascriptSIMDInt8x16>(args[1], args[2], 8 * TySize[TyInt16], scriptContext);
+        return SIMD128TypedArrayLoad<JavascriptSIMDInt8x16>(args[1], args[2], 16 * INT8_SIZE, scriptContext);
```

Comment:
This change is intentional and the new usage is more appropriate. This wasn't a logic bug as 8*16 and 16*8 computes the same product and has no logical difference.
This just makes the size computation macro(name) usage consistent across the other simd types.